apps/courses/src/components/CircledNumber.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
```

Comment:
I doubt that this component justifies a file of its own. Put this within the context of the file in which its used.